DataFormats/L1TrackTrigger/interface/TTBV.h

```diff
@@ -103,39 +122,49 @@ class TTBV {
     bs_ ^= bv.bs_;
     return *this;
   }
+
+  // operator: not
   TTBV operator~() const {
     TTBV bv(*this);
     return bv.flip();
   }
 
-  // operator: bit shifts
+  // operator: bit shifts right reference
   TTBV& operator>>=(int pos) {
     bs_ >>= pos;
     size_ -= pos;
     return *this;
   }
+
+  // operator: bit shifts left reference
   TTBV& operator<<=(int pos) {
     bs_ <<= S - size_ + pos;
     bs_ >>= S - size_ + pos;
     size_ -= pos;
     return *this;
   }
+
+  // operator: bit shifts left copy
   TTBV operator<<(int pos) const {
     TTBV bv(*this);
     return bv >>= pos;
   }
+
+  // operator: bit shifts right copy
   TTBV operator>>(int pos) const {
     TTBV bv(*this);
     return bv <<= pos;
```

Comment:
Comment says this is bit shift right. So why is left shift operator used?

Reply:
done